app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post
+  after_save :update_comments_counter
+
+  def update_comments_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ ] **[OPTIONAL]** I think it would be a good idea to make this method private